### PR TITLE
Feat/begin styles

### PR DESCRIPTION
**Pull request by Krzysztofe**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task goal
The task was to create a splash screen based on the provided design. The UI should be as close as possible to the design.

### Review

File: src/components/circle/Circle.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import { Context } from "../../context/ContextProv";
+
+type Props = {
+  children?: ReactNode;
+  width: string;
+  translate: string;
+  keyFrames: { name: string; transformTo: string; scaleTo: string };
+};
+
+const css = (width: string, translate: string, name: any) => {
```
Comment (Sumick) [inline src/components/circle/Circle.tsx:11]: **nitpick**: We should not leave any any

File: src/components/circle/Circle.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      100% {
+        transform: translate(${translateTo}) scale(${scaleTo});
+      }
+    }
+  `;
+};
+
+const Circle = (props: Props) => {
+  const { isBegin } = useContext(Context);
+  const keyframes = generateKeyframes(props.keyFrames);
```
Comment (Sumick) [inline src/components/circle/Circle.tsx:42]: **praise:** Nice, and simple!

**Review by Sumick — APPROVED** [review]
Very nice, and readable code. Good job! Im leaving 2 Typescript related nits to fix :)

**Pull request merged**